linkerd/docs/namer.md

```diff
@@ -699,8 +699,8 @@ would be rewritten to `/pfx/foo/resource/name`
 ```
 
 Rewrites a name of the form "host:ip" as a path with host followed by ip. Does
-not not support IPv6 host IPs (because ipv6 notation doesn't work in Paths as-
-is, due to bracket characters).
+not not support IPv6 host IPs (because ipv6 notation doesn't work in Paths as-is
```

Comment:
I think there's another typo here: "not not". Can you also standardize the capitalization of IPv6?